lunatrace/bsl/license-worker/internal/pkg/scanner/test/package/cjs/react-jsx-dev-runtime.profiling.min.js

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Do these files need to get nuked?

Reply:
These are test files, I could make them shorter but I think they're good for testing if header scanning works.